lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -290,9 +290,17 @@ struct RvalueExprVisitor {
     }
 
     case BinaryOperator::Equality:
-      return createBinary<moore::EqOp>(lhs, rhs);
+      if (isa<moore::UnpackedArrayType>(lhs.getType()))
+        return builder.create<moore::UArrayCmpOp>(
+            loc, circt::moore::UArrayCmpPredicate::eq, lhs, rhs);
```

Comment:
You can use the `moore` namespace directly here:
```suggestion
            loc, moore::UArrayCmpPredicate::eq, lhs, rhs);
```